Frends.PowerShell.RunScript/Frends.PowerShell.RunScript/Frends.PowerShell.RunScript.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+	<TargetFrameworks>net6.0</TargetFrameworks>
+    <authors>HiQ Finland</authors>
+    <copyright>HiQ Finland</copyright>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+	<PackageProjectUrl>https://github.com/FrendsPlatform/Frends.PowerShell</PackageProjectUrl>
+    <IncludeSource>true</IncludeSource>
+    <PackageTags>Frends</PackageTags>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <Version>1.0.1</Version>
+	<Description>Task for running scripts on PowerShell. Requires PowerShell 7.0 to work.</Description>
+  </PropertyGroup>
```

Comment:
Please double check Frends Task guideline for package information and modify the package information to match the guidelines. Here is an example: https://github.com/FrendsPlatform/Frends.Excel/blob/main/Frends.Excel.ConvertToCSV/Frends.Excel.ConvertToCSV/Frends.Excel.ConvertToCSV.csproj

Reply:
Fixed